qc/workflow_exome.cwl

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: Workflow
+label: "Exome QC workflow"
+requirements:
+    - class: SubworkflowFeatureRequirement
+inputs:
+    bam:
+        type: File
+        secondaryFiles: [.bai]
+    reference:
+        type: File
+        secondaryFiles: [.fai]
+    bait_intervals:
+        type: File
+    target_intervals:
+        type: File
+    roi_intervals:
```

Comment:
I'm totally fine with changing this so that select_variants/verify_bam_id runs on the target_intervals instead.